crates/metaslang/cst/src/query/parser.rs

```diff
@@ -9,14 +9,14 @@ use nom::multi::{fold_many0, many0, many1};
 use nom::sequence::{delimited, pair, preceded, terminated};
 use nom::{Finish, IResult, Parser};
 
-// This crate is copied to another crate, so all imports should be relative
-use super::super::kinds::{NodeLabel, RuleKind, TokenKind};
 use super::model::{
     AlternativesMatcher, BindingMatcher, Kind, Matcher, NodeMatcher, NodeSelector,
     OneOrMoreMatcher, OptionalMatcher, SequenceMatcher,
 };
+// This crate is copied to another crate, so all imports should be relative
```

Comment:
IIUC, this is no longer copied. Should we remove this left-over comment?

Reply:
done

Reply:
I think it still exists in 3 other places.